matplotlib_scalebar/dimension.py

```diff
@@ -25,6 +25,7 @@
     "\u00b5": 1e-6,
     "u": 1e-6,
     "n": 1e-9,
+    "A": 1e-10,
```

Comment:
Angstroms is not a prefix. It should not appear in this list.

Reply:
Just wondering how to convert between Angstrom and other units then if not using the `_PREFIXES_FACTORS`. Create a sperate scaling factors dictionary? and put it a new `LengthDimension` as the child of  `_Dimension`? Given that except Angstrom all the other units are well covered, so it will be a class with one case only.